templates/robotics/ros-esm.html

```diff
@@ -44,203 +38,177 @@ <h4>
-        What is ROS ESM?
-      </h2>
-      <p>
-        As your code develops and packages change, ROS needs to be continuously patched and updated, along with your other software dependencies. This time-consuming endeavour detracts from your robotics development. Duplicating and freezing upstream repositories is not a secure solution. Running un-patched and unmaintained versions of ROS exposes your robot, company and customers to serious risk. The challenge increases when working with End-Of-Life (EOL) ROS distributions or dependencies such as ROS Kinetic and Python 2.
-      </p>
-      <p>
-        Delivered in partnership with Open Robotics, ROS ESM provides a hardened and long-term supported ROS environment for your robot and its applications. With ROS ESM, you can count on backports for critical security updates, common vulnerabilities and exposures (CVE) fixes and bug fixes for ROS and Ubuntu base OS distribution.
-      </p>
-      <p>
-        <a href="/blog/what-is-ros-esm">
-          Learn more about ROS ESM&nbsp;&rsaquo;
-        </a>
-      </p>
+<section class="p-strip is-x-deep u-no-padding--bottom">
+  <div class="p-strip u-no-padding--top">
+    <div class="row">
+      <div class="col-6 col-medium-3">
+        <h2>What is ROS ESM?</h2>
+      </div>
+      <div class="col-6 col-medium-3">
+        <p>Whether you want to be compliant with cybersecurity regulations, win more time before your migration, or reduce operational expenses of maintaining upstream repositories, ROS ESM is here to make your work easier.</p>
+        <p>Robot Operating System Expanded Security Maintenance (ROS ESM) is a service by Canonical that provides security maintenance for ROS Long Term Support (LTS) releases and the underlying Ubuntu distributions, starting with ROS Kinetic.</p>
+        <p>This includes the underlying Ubuntu system and more than 23,000 packages from the Ubuntu Universe repository.</p>
+        <p>ROS ESM is available with your <a href="/pro">Ubuntu Pro subscription</a>.</p>
+      </div>
     </div>
-    <div class="col-4 u-align--center u-vertically-center">
-      <div>
-
-        <div class="p-card u-align--left">
-          <h3 class="p-muted-heading">
-            Delivered in partnership with
-          </h3>
-          <hr class="u-sv1">
-          <div>
-            <a href="https://www.openrobotics.org/">
-              {{
-                image(
-                url="https://assets.ubuntu.com/v1/8da00274-open-robotics.png",
-                alt="",
-                height="61",
-                width="214",
-                hi_def=True,
-                attrs={ "style": "margin: 1rem 0;"},
-                loading="lazy",
-                ) | safe
-              }}
-            </a>
-          </div>
-          <p class="p-card__content">
-            Maintainer of the Robot Operating System and Gazebo
-          </p>
+  </div>
+  <hr class="is-fixed-width">
+  <div class="p-strip">
+    <div class="u-fixed-width">
```

Comment:
this div isn't needed actuallly - removing it will remove the bug described in the previous comment